tasks/mc/data.py

```diff
@@ -364,19 +413,28 @@ def _insert_tr(self, session):
         session.execute(query)
         session.commit()
 
+    PK_EXISTS_RE = '.*multiple primary keys for table \".*\" are not allowed.*'
+
     def _create_constraints(self, session):
-        LOGGER.info('Creating constraints for {}'.format(self.output().table))
+        output = self.output()
+        LOGGER.info('Creating constraints for {}'.format(output.table))
 
         query = '''
                 ALTER TABLE "{schema}".{table}
                 ADD PRIMARY KEY({region_id}, {month});
                 '''.format(
-                    schema=self.output().schema,
-                    table=self.output().tablename,
+                    schema=output.schema,
+                    table=output.tablename,
                     region_id=REGION_ID_COLUMN[1],
                     month=MONTH_COLUMN[1],
                 )
-        session.execute(query)
+        try:
+            session.execute(query)
+        except Exception as e:
+            if re.match(self.PK_EXISTS_RE, str(e)):
```

Comment:
Probably better to match by [error code](https://www.postgresql.org/docs/10/errcodes-appendix.html) which is accessible via `e.orig.pgcode` (`e` is SQLAlchemy's exception, `e.orig` is PsycoPG2's exception)